source/notebooks/higgsto4l.ipynb

```diff
@@ -27,8 +27,8 @@
    "source": [
     "%matplotlib inline\n",
     "import numpy as np\n",
-    "import awkward1 as ak\n",
-    "import awkward\n",
+    "import awkward as ak\n",
+    "import awkward0\n",
```

Comment:
Is moving to awkward1 required for binder? If this is the case, I feel like we should prioritize keeping the interface as up to date as possible over having an binder example

Reply:
Yes, this move is for binder. I find some version conflicts. ok, maybe I can close this PR.